docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md

```diff
@@ -0,0 +1,84 @@
+# Gas fee payment in WBTC using ERC-2771
```

Comment:
```suggestion
# Gas fee payment in WBTC using OpenGSN & ERC-2771
```

ERC-2771 is only a small part of opengsn - I would put a little bit more emphasis on the opengsn part, also in the into below

Reply:
Please also use the common names in the heading - Meta Transactions